server/src/index.ts

```diff
@@ -62,13 +62,7 @@ async function main() {
   app.use(globalRateLimit)
   app.use(sessionMiddleware)
   app.use(passport.session())
-  app.use(
-    '/api-docs',
-    swaggerUI.serve,
-    swaggerUI.setup(swaggerDocument, {
-      withCredentials: true,
-    })
-  )
+  app.use('/api-docs', swaggerUI.serve, swaggerUI.setup(swaggerDocument))
```

Comment:
Why no need specifying withCredentials?

Reply:
Because swagger doesn't have that option in fact.